### ISSUE TITLE
latest pgwatch2 gives http error 404 (not found) for localhost:8086 - InfluxDB API

### ISSUE THREAD

**Issue by georg-h** [cybertec-postgresql/pgwatch2#28]
Hello Kaarel,
after getting latest pgwatch2 I found that InfluxDB API is giving http error 404 (not found) when opening http://localhost:8086 (this worked in my previous version)
The port is included in my docker run xxx command:
-p 127.0.0.1:8083:8083 -p 127.0.0.1:8086:8086 -p 127.0.0.1:8088:8088
but doesn't work anymore. Nevertheless netstat -ln shows this port as listening on the host

]$ netstat -ln
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 0.0.0.0:111             0.0.0.0:*               LISTEN
tcp        0      0 x.x.x.x:8080             0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:8083          0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:8086          0.0.0.0:*               LISTEN
tcp        0      0 10.192.64.101:3000      0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:5432          0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:8088          0.0.0.0:*               LISTEN

**Comment by kmoppel** [comment]
Hi! Could not replicate the problem myself...but looking at the InfluxDB commit log I think there might been a bug introduced in the last pgwatch version (1.3.5). Will investigate a bit and push new images soon.

**Comment by kmoppel** [comment]
New image now available - see if it helps. When not, one should get some info from InfluxDB logs.

**Comment by georg-h** [comment]
Hello Karel,
sorry for the late reply. On wednesday I had to recreate the docker container as we had to switch docker to use direct-lvm mode instead of loop-lvm mode due to "no space left on device" within the loop-file.
So I pulled pgwatch2 fresh from docker registry but the problem with the InfluxDB API  still persists.

I've checked the influx log but there are not suspicious messages regarding this interface. The port on the docker host is still listen ( tcp 0 0 127.0.0.1:8086 0.0.0.0:* LISTEN ) but opening http://127.0.0.1:8086 gives me a http error 404.
Do I have to append something behind this link? e.g. a certain page?

**Comment by kmoppel** [comment]
Yes, a certain page is expected on Influx side I think. Could you try: ```curl http://127.0.0.1:8086/query --data-urlencode q="show databases"```
The query endpoints are described in detail here: https://docs.influxdata.com/influxdb/v1.4/guides/querying_data/

**Comment by georg-h** [comment]
Hello Karel, 
I've found now what's wrong (I was it too with my ticket description)
What I missed is the Web Admin Interface of InfluxDB (originally listen on 8083) and I found now that this is deprecated and disabled by default sind influxDB v1.1
https://docs.influxdata.com/influxdb/v1.2/tools/web_admin/

so I close this issue and will use influx cmd
